server/routes.js

```diff
@@ -409,7 +409,8 @@
             var prj = project_path.split("server/../");
             var file_path = home_dir(current_user(req)) + project_name + "/" +
                             file_name;
-            execOnServer('mkdir ' + home_dir(current_user(req)) + project_name,
+            execOnServer('rm -rf ' + home_dir(current_user(req)) + project_name +
```

Comment:
Being a 'rf -rf' shouldn't we take extra care to ensure that nothing bad happens?